receivers/alertmanager/alertmanager.go

```diff
@@ -77,23 +70,23 @@ func buildAlertmanagerNotifier(fc FactoryConfig) (*AlertmanagerNotifier, error)
 	}, nil
 }
 
-// AlertmanagerNotifier sends alert notifications to the alert manager
-type AlertmanagerNotifier struct {
-	*Base
-	images   ImageStore
+// Notifier sends alert notifications to the alert manager
+type Notifier struct {
```

Comment:
We called the package `receivers` but generalized all the notifier types `Notifier` - WDYT on aligning this? I.e. calling the package two layers up `notifiers` or calling these structs `Receiver`?

Do "notifier" and "receiver" always mean the same thing in our context?

Reply:
The reason I chose package `receivers` and struct name `Notifier` is that:
- we have `notify` package, and having `notifiers` would look awkward. If we want to rename to notifiers then probably we need to rename `notify` package.
- Notifier implements upstream's interface Notifier that has method Notify. Receiver in Alertmanager terminology is a set of Integrations, where each Integration wraps a Notifier.